### PR TITLE
[Docs] Update mmcls doc

### PR DESCRIPTION
**Pull request by Renzhihan**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily receiving feedbacks. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit tests to ensure the correctness.
3. If the modification has a dependency on downstream projects of a newer version, this PR should be tested with all supported versions of downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

**Comment by Renzhihan** [comment]
I am not sure what to fill in for PPLNN in the readme. I have kept it consistent with before. Please give me some advice on how to improve it.

**Review by RunningLeon — COMMENTED** [review]
If you have time, you could install pplnn and check if the models could be deployed on pplnn successfully. If not, could keep them as original.
Besides, you could update https://github.com/open-mmlab/mmdeploy/blob/main/docs/zh_cn/04-supported-codebases/mmcls.md

File: docs/zh_cn/04-supported-codebases/mmcls.md [hunk 1]
```diff
@@ -187,3 +187,6 @@ for label_id, score in result:
 | [ShuffleNetV2](https://github.com/open-mmlab/mmclassification/tree/1.x/configs/shufflenet_v2)           |      Y      |      Y       |    Y     |  Y   |   Y   |    Y     |
 | [VisionTransformer](https://github.com/open-mmlab/mmclassification/tree/1.x/configs/vision_transformer) |      Y      |      Y       |    Y     |  Y   |   ?   |    Y     |
 | [SwinTransformer](https://github.com/open-mmlab/mmclassification/tree/1.x/configs/swin_transformer)     |      Y      |      Y       |    Y     |  N   |   ?   |    N     |
+| [MobileOne](https://github.com/open-mmlab/mmclassification/tree/1.x/configs/mobileone)                  |      Y      |      Y       |    N     |  N   |   ?   |    N     |
+| [EfficientNet](https://github.com/open-mmlab/mmpretrain/tree/main/configs/efficientnet)                 |      Y      |      Y       |    N     |  N   |   ?   |    N     |
+| [Conformer](https://github.com/open-mmlab/mmpretrain/tree/main/configs/conformer)                       |      Y      |      Y       |    N     |  N   |   ?   |    N     |
```
[the remark this answers is not in the document]
Comment (Renzhihan) [reply docs/zh_cn/04-supported-codebases/mmcls.md:192]: OK.  In addition, I have another confusion. Some models in md files are not found in the yml file, for example VisionTransformer. This situation also occurs in mmseg.md.

[the remark this answers is not in the document]
Comment (RunningLeon) [reply docs/zh_cn/04-supported-codebases/mmcls.md:192]: Could test and add to yml file if supported. We plan to generate the supported model table automatically from yml file later.

**Review by RunningLeon — APPROVED** [review]
LGTM

**Review by lvhan028 — APPROVED** [review]

**Pull request merged**